### DISCUSSION TITLE
Create PR API returning 502 - Bad Gateway

### DISCUSSION THREAD

**Discussion by sameer-coder** [community/community#43200] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

Hello,

I am trying to create a PR on one of my repositories and getting a response from the GitHub API with status code `502`  and the message `Server Error`. The endpoint is `https://api.github.com/repos/owner/repo/pulls`. The length of the PR body is 61448 characters. The problem is reproducible both using an HTTP client like POSTMAN and the GitHub SDK. Also, it would be good to know what is the _maximum limit of characters allowed in the PR body when creating a PR using the API_.

I haven’t found any potential solutions to this in the documentation. Any help would be greatly appreciated!

Thanks!

**Comment by wiebeck** [comment]
Has your issue been resolved? What was the cause if I may ask?

**Reply by simoneb** [reply]
We trimmed the body to a maximum number of characters
